provider/filestore.go

```diff
@@ -299,7 +300,7 @@ func NewGCSFileStore(config Config) (FileStore, error) {
 		return nil, fmt.Errorf("could not deserialize config: %v", err)
 	}
 
-	creds, err := google.CredentialsFromJSON(context.TODO(), GCSConfig.Credentials, "https://www.googleapis.com/auth/cloud-platform")
+	creds, err := google.CredentialsFromJSON(context.TODO(), GCSConfig.Credentials.Credentials, "https://www.googleapis.com/auth/cloud-platform")
```

Comment:
Should probably be changed to GCSConfig.Config.Credentials

Reply:
so the struct is 

```
type GCSFileStoreConfig struct {
	BucketName  string
	BucketPath  string
	Credentials GCPCredentials
}
```
should I modify the Credentials key to `Config`?

Reply:
Yeah since it includes more than just credentials, makes sense to rename it